### ISSUE TITLE
tooltip can not be 'false'

### ISSUE THREAD

**Issue by Bommel24** [xwpongithub/vue-range-slider#20]
The tooltip validator only returns true for 'always' and 'hover'. You can't set the tooltip to 'false' as stated in the readme.

**Comment by rftahir** [comment]
It's already updated at https://github.com/xwpongithub/vue-range-slider/pull/9
use `:tooltip=false` instead of `:tooltip="false"` or `tooltip="false"`

**Comment by basil-gor** [comment]
Yes. I have the same problem. Temporary solved it by set `tooltip-style: {display: "none",}`
